### PR TITLE
support of login screen added to publications

### PR DESCRIPTION
**Pull request by xkureck**

### CODE REVIEW

**Review by Vojtech-Sassmann — CHANGES_REQUESTED** [review]
The implementation looks good I am just wondering if we need to duplicate the `publications-login-screen-base` component? I think it would be better to use the already existing `perun-web-apps-login-screen-base` component and if you want the book icon to appear, you can add some input parameter which can be set.

File: libs/perun/services/src/lib/auth.service.ts [hunk 1]
```diff
@@ -72,13 +72,6 @@ export class AuthService {
     this.manager.events.addUserLoaded(user => {
       this.user = user;
     });
-
-    this.manager.events.addAccessTokenExpired(() => {
-      const config = getDefaultDialogConfig();
-      config.width = '450px';
-
-      this.dialog.open(SessionExpirationDialogComponent, config);
-    });
```
Comment (xkureck) [inline libs/perun/services/src/lib/auth.service.ts:81]: check this. Was it there for some reason? Is it safe to delete it?

Comment (Vojtech-Sassmann) [reply libs/perun/services/src/lib/auth.service.ts:81]: I think so. Probably, it is used to display a dialog when the access token has expired. It shouldn't be probably removed.

**Review by Vojtech-Sassmann — CHANGES_REQUESTED** [review]

**Review by Vojtech-Sassmann — CHANGES_REQUESTED** [review]
Please, keep the SessionExpirationDialogComponent. Otherwise, I think these changes are ok.

**Review by Vojtech-Sassmann — APPROVED** [review]

**Pull request merged**